package.json

```diff
@@ -185,6 +187,28 @@
       "pre-commit": "npm run lint"
     }
   },
+  "bundlesize": [
+    {
+      "path": "./static/vendor*.js",
+      "maxSize": "200 kB"
+    },
+    {
+      "path": "./static/[0-9].*.js",
+      "maxSize": "20 kB"
+    },
+    {
+      "path": "./static/[1-9].*.css",
+      "maxSize": "2.1 kB"
```

Comment:
There is a file with 2.08kB, near the limit. We should grow up to 2.5kB

Reply:
I was trying to be picky 😆